github-for-jira.sd.yml

```diff
@@ -101,20 +118,18 @@ environmentOverrides:
         INSTANCE_NAME: github-for-jira-dev
         NODE_OPTIONS: "--no-deprecation"
         LOG_LEVEL: debug
-        # * Commented out env vars are to be saved in secrets
-        # WEBHOOK_SECRET: for signed signature when syncing an org
-        # MICROS_ENV: micros environment for deployment
-        # MICROS_ENVTYPE: the type of micros environment for deployment
+        SENTRY_ENVIRONMENT: ddev
+        APP_ID: '124403'
+        GITHUB_CLIENT_ID: Iv1.600bf90a20f1ab18
```

Comment:
Moved all non-secret environment variables into the service descriptor for better visibility.